qml/MonthComponent.qml

```diff
@@ -338,7 +338,14 @@ Item{
             id: weekDay
             objectName: "weekDay" + index
             width: parent.dayWidth
-            property var day : Qt.locale(Qt.locale().name).standaloneDayName(( Qt.locale().firstDayOfWeek + index), Locale.ShortFormat);
+            property var day : {
+		        if (Qt.locale().name == "pt_PT") {
+                    // Workaround for https://github.com/ubports/ubuntu-touch/issues/510
+                    Qt.locale("pt").standaloneDayName(( Qt.locale().firstDayOfWeek + index), Locale.ShortFormat);
```

Comment:
Nitpick: When defining property bindings in `{ }` you should use the `return` statement . See the second example [here](https://doc.qt.io/qt-5/qtqml-syntax-propertybinding.html)